### DISCUSSION TITLE
지난 회의에서 확정 한 구현 기능이 일부 모호해서 여러분 생각을 여쭤봅니다.

### DISCUSSION THREAD

**Discussion by greyHairChooseLife** [wanted-pre-onboarding-side-project/habit-tracker#39] [Q&A]
위키에 정리한 회의록의 [이 부분](https://github.com/wanted-pre-onboarding-side-project/habit-tracker/wiki/%ED%9A%8C%EC%9D%98%EB%A1%9D#3habit-%EB%A6%AC%EC%8A%A4%ED%8A%B8) 보시면 아래와 같은 내용이 있습니다.

>- 지나간 요일도 체크/해제 가능

이때 지나간 요일이라는 것은 금주에 한정이겠지요? 그러니까 지난주나 지 지난주는 체크 한 것을 수정 할 수 없는 것이겠지요?

제 생각은 그러한데 혹시 달리 생각하실 수도 있어서 확실하게 하고자 여쭤봅니다! 

`Q&A` 카테고리이긴 하지만 다들 이견이 없다면 해당 위키 문구를 아래와 같이 수정 해 두겠습니다.

'지나간 요일도 체크/해제 가능' :point_right: '지나간 요일도 체크/해제 가능(금주에 한정)'

**Comment by jiyeon2** [comment]
저는 이번주, 지난주 상관없이 지나간 일자도 체크를 수정할 수 있다고 생각했습니다~
하지만 금주에 한정해도 괜찮을거같아요!

**Comment by chadseok** [comment]
네 저도 모든 지나간 일자에 수정이 가능하다고 생각했습니다. 그리고 저는 금주에만 한정해야 하는 것에 대한 특별한 이유가 없다면 모든 지나간 일자에 대해 수정가능해야 한다고 생각합니다

혹시 @greyHairChooseLife 님이 금주에 한정해야 한다고 생각하신 이유에 대해서 알 수 있을까요?

**Reply by greyHairChooseLife** [reply]
네, 왜냐하면 (시간이)지나간 것들을 수정 가능하다면 tracking이 아니기 때문입니다.
   
track은 동사로는 추적한다, 명사로는 머무른 자취나 지나간 자국을 뜻합니다. 과거를 추적 해 가는데, 관련 된 기록들을 수정하는 것은 앞뒤가 맞지 않는 행동, 그래서 추적(tracking)이 아무 의미가 없는 행동 됩니다.
